server/data/UseOfForceReport.ts

```diff
@@ -0,0 +1,60 @@
+export type IncidentDetails = {
```

Comment:
created a type to represent the complete form data. This is mostly right I think. 

Could look at trying to represent values which are codes of enums as a restricted set of values